metadata-ingestion/setup.py

```diff
@@ -829,6 +829,7 @@
         "glue = datahub.ingestion.source.aws.glue:GlueSource",
         "sagemaker = datahub.ingestion.source.aws.sagemaker:SagemakerSource",
         "hana = datahub.ingestion.source.sql.hana:HanaSource",
+        "hbase = datahub.ingestion.source.sql.hbase:HBaseSource",
```

Comment:
additional to the entry point, hbase dependencies should be set here

https://github.com/btkcodedev/datahub/blob/85649af65c0f1e3068d68633b3e983994710eea2/metadata-ingestion/setup.py#L404-L405